spec/lib/spotlight/dor/indexer_spec.rb

```diff
@@ -222,38 +206,33 @@
 
     describe '#add_general_notes' do
       before do
-        allow(r).to receive(:mods).and_return(mods)
         subject.send(:add_general_notes, sdb, solr_doc)
       end
 
       context 'no general notes, but other types of notes' do
-        let(:mods) do
-          Nokogiri::XML <<-EOF
-            <mods xmlns="#{Mods::MODS_NS}">
-              <note displayLabel="preferred citation">(not a document subtype)</note>
-              <note displayLabel="Document subtype">memorandums</note>
-              <note displayLabel="Donor tags">Knowledge Systems Laboratory</note>
-            </mods>
-            EOF
+        let(:modsbody) do
+          <<-EOF
+            <note displayLabel="preferred citation">(not a document subtype)</note>
+            <note displayLabel="Document subtype">memorandums</note>
+            <note displayLabel="Donor tags">Knowledge Systems Laboratory</note>
+          EOF
         end
 
         it 'is blank' do
-          expect(solr_doc['general_notes_ssim']).to be_blank
+          expect(solr_doc).not_to include 'general_notes_ssim'
         end
       end
 
       context 'ignore extra notes' do
-        let(:mods) do
-          Nokogiri::XML <<-EOF
-            <mods xmlns="#{Mods::MODS_NS}">
-              <note displayLabel="Document subtype">memorandums</note>
-              <note>a generic note</note>
-            </mods>
-            EOF
+        let(:modsbody) do
+          <<-EOF
+            <note displayLabel="Document subtype">memorandums</note>
+            <note>a generic note</note>
+          EOF
         end
 
         it 'extracts the doc subtypes' do
-          expect(solr_doc['general_notes_ssim']).to contain_exactly 'a generic note'
```

Comment:
For a single string in an array, no real benefit.  Value here really is expected to be an Array, not just some Enumerable (because that's what solr will accept).  I like many of the advanced rspec matchers, but I found this more accurate/readable.  If I were going to use one, it would be something like:
```ruby
expect(solr_doc).to match a_hash_including('general_notes_ssim' => ['a generic note'])
expect(solr_doc).to include('general_notes_ssim' => ['a generic note']) # or simply this
```

Because that helps when fields drift or get renamed: instead of just saying `expected "a generic note", got nil`, it shows what *is* in the hash.  And if I was going to do that, I would be changing the majority of examples in the file.